SalesDataPage/JS/app.js

```diff
@@ -0,0 +1,98 @@
+var hoursOpen = ['10am', '11am','12pm','1pm','2pm','3pm','4pm','5pm','6pm'];
+var storeArr = [];
+
+function Store(storeName, minCust, maxCust, avgCookie){
+  this.storeName = storeName;
+  this.minCust = minCust;
+  this.maxCust = maxCust;
+  this.avgCookie = avgCookie;
+  storeArr.push(this);
+}
+Store.prototype.hourlySales = [];
+Store.prototype.dailyTotal = 0;
+Store.prototype.getSalesNums = function(minNum, maxNum){
+  return(Math.random() * (maxNum - minNum)) + minNum;
+};
+Store.prototype.totals = function(rand, maxN, minN, avgC, hour, hourlyS, dayTotal) {
+  for(var s = 0; s < hour.length; s++) {
+    this.hourlySales[s] = Math.floor(rand(minN, maxN) * avgC);
+    this.dailyTotal += this.hourlySales[s];
+  }
+};
+Store.prototype.render = function() {
+  this.totals(this.getSalesNums, this.maxCust, this.minCust, this.avgCookie, hoursOpen, this.hourlySales, this.dailyTotal);
+  storeTitle(this.storeName, this.hourlySales, this.dailyTotal, hoursOpen);
+};
+
+function storeTitle(nameS, hourlyS, dayTotal, hours) {
```

Comment:
Tip: try to be more descriptive in naming your variables. Your arguments `nameS` and `hourlyS` are kinda cryptic.